### ISSUE TITLE
immunization notification/reminder system

### ISSUE THREAD

**Issue by Kyle-Falconer** [codeforsanjose/Project-Ideas#83]
India has this reminder system that works over SMS. Here's the description for it:

*Request all parents to use this brilliant service*

One more initiative by our PM Narendra Modi for a healthy future generation
*National Vaccine Reminder* is a free service.

Just SMS your child's name and date of birth to *566778* in the following format:-

*Immunize*[space] *child's name* [space] child's *date of birth*

Example - *Immunize Krishna 01-01-2015*

Reminder for every vaccination of the child will be SMSed to you 2 days in advance.

This will continue till the time your child is 12 years old.

*Gift your child a healthy life..*

Pl forward this msg to all your friends in your contacts.

*ALL WHO HAS KIDS BELOW 12 YEARS MUST REGISTER THEMSELVES*
Kindly circulate to all paediatrician.

**Comment by antoineblueberry** [comment]
@Kyle-Falconer Sorry to revive this. I really like this idea and what India has done. Has anyone taken on this project?

**Comment by Kyle-Falconer** [comment]
@antoineblueberry Nope! It's free at the moment, though if you want to work on this together, I'd be down for doing what I can.

I imagine it shouldn't be too bad to do purely over SMS using [Twilio](https://www.twilio.com/sms).

Also, there is an abandoned project around tracking immunizations here: https://github.com/codeforsanjose/ImmunizationSJ
